pkg/virt-controller/watch/drain/evacuation/evacuation.go

```diff
@@ -553,7 +554,9 @@ func (c *EvacuationController) numOfVMIMForThisSourceNode(
 
 	for _, vmim := range activeMigrations {
 		if _, ok := vmiMap[vmim.Spec.VMIName]; ok {
-			activeMigrationsFromThisSourceNode++
+			if vmim.IsRunning() {
+				activeMigrationsFromThisSourceNode++
+			}
```

Comment:
Couldn't this function be called with the list filtered by `FilterRunningMigrations()` and avoid checking `vmim.IsRunning()`?

Reply:
@jean-edouard Thank you! done